### DISCUSSION TITLE
Is there any reason the frontend never gets an updates or anything new

### DISCUSSION THREAD

**Discussion by cloudbr34k84** [Koenkk/zigbee2mqtt#19497] [General]
For example
Creating Groups would be better if we could just search for everything by string of text, then select all we want to add to the group.

or could we not see all the Devices connected to a router decice when viewing that device's page??

If I could help I would, but don't know a thing about coding
Anyway just a discussion that's all

**Comment by DubhAd** [comment]
You could open a feature request on the [frontend repo](https://github.com/nurikk/zigbee2mqtt-frontend) - and yes, there's updates to that all the time ;)
